package.json

```diff
@@ -42,7 +44,8 @@
     "lint": "standard",
     "postinstall": "lerna bootstrap",
     "doc:lint": "remark .",
-    "start": "node packages/udaru-server/start.js",
+    "start-v16": "node packages/hapi-auth-udaru-16/start.js",
+    "start": "node packages/hapi-auth-udaru/start.js",
```

Comment:
discussed on slack, but let's have one start, pointing to node packages/hapi-auth-udaru-16/start.js